src/simply/simply_stage.c

```diff
@@ -618,6 +664,12 @@ bool simply_stage_handle_packet(Simply *simply, Packet *packet) {
     case CommandElementRadius:
       handle_element_radius_packet(simply, packet);
       return true;
+    case CommandElementAnglestart:
+      handle_element_anglestart_packet(simply, packet);
```

Comment:
`anglestart` should be `angle_start` since they are two terms. The struct member fields and related will have to be updated.